build.zig

```diff
@@ -17,10 +17,54 @@ pub fn build(b: *std.Build) !void {
         },
     };
 
+    const c_sdl_preferred_linkage = b.option(
+        std.builtin.LinkMode,
+        "c_sdl_preferred_linkage",
+        "Prefer building statically or dynamically linked libraries (default: static)",
+    ) orelse .static;
+    const c_sdl_strip = b.option(
+        bool,
+        "c_sdl_strip",
+        "Strip debug symbols (default: varies)",
+    ) orelse (optimize != .Debug);
+    const c_sdl_sanitize_c = b.option(
+        enum { off, trap, full }, // TODO: Change to std.zig.SanitizeC after 0.15
+        "c_sdl_sanitize_c",
+        "Detect C undefined behavior (default: varies)",
+    ) orelse .off;
+    const c_sdl_pic = b.option(
+        bool,
+        "c_sdl_pic",
+        "Produce position-independent code (default: varies)",
+    ) orelse true;
```

Comment:
The computed default value for this one is pretty complicated. I would suggest not exposing the option at all, because using the wrong default *will* break the build for a lot of targets. The rare few users who really need to override the default can probably find a way to do so themselves by messing with some build system internals.

https://github.com/ziglang/zig/blob/e96d86064eb81977f254fa8f36481b7d150cb3b6/src/Package/Module.zig#L156-L175

https://github.com/ziglang/zig/blob/e96d86064eb81977f254fa8f36481b7d150cb3b6/src/target.zig#L47-L52